Automattic/wp-calypso
# Update the no backup view to match with the design

#### Changes proposed in this Pull Request
**Asana:** 1142395350490785-as-1173737050079882

It fixes the styles in the Backup view for the "No Backup" status. Added the correct icon. Also, this PR extracts the pattern for the same views, so we can reuse it and apply to others.

Before:
![image](https://user-images.githubusercontent.com/9832440/82040629-c7d75a80-969e-11ea-8b82-05abc23c22ab.png)
![image](https://user-images.githubusercontent.com/9832440/82040652-d45bb300-969e-11ea-8c85-2340fb086931.png)


After:
![image](https://user-images.githubusercontent.com/9832440/82040547-a70f0500-969e-11ea-8865-d24a04086288.png)
![image](https://user-images.githubusercontent.com/9832440/82040581-b3935d80-969e-11ea-8b85-c939678d697e.png)


#### Testing instructions

- Apply the changes
- Check if it matches with the expected design

#### Next

- Fix the rest of the Backup status view by adding the correct icon and applying the common classes for the Backup status views.
